### ISSUE TITLE
[Improvement] Internationalization

### ISSUE THREAD

**Issue by Vector-SMG** [Kong/insomnia#1541]
- Insomnia Version: ___
- Operating System: ___

**Comment by gschier** [comment]
I think internationalization is a great idea as Insomnia gets more popular.

Can you elaborate on what parts of the app should be covered first?
